packages/itwinui-css/src/transfer-list/transfer-list.scss

```diff
@@ -10,10 +9,6 @@
   column-gap: var(--iui-size-xs);
 }
 
-.iui-transfer-list-listbox-label {
```

Comment:
make sure to document this breaking css change

Reply:
Documented in changeset